apps/hubble-stats/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -10,50 +10,43 @@ import {
 import { CoinIcon, ContrastLine } from '@webb-tools/icons';
 
 import { VANCHORS_MAP } from '../../constants';
-
 import { BreadcrumbType } from './types';
 
 const Breadcrumbs: FC = () => {
   const pathname = usePathname();
   const parts = pathname.split('/');
-  let activeItem = parts[parts.length - 1];
 
-  // check if current path is /pool/<poolAddress>
-  if (parts.length === 3 && parts[0] === '' && parts[1] === 'pool') {
-    const poolAddress = parts[2];
-    // if invalid poolAddress, the breadcrumb only show the default item
-    if (!VANCHORS_MAP[poolAddress]) {
-      activeItem = '';
-    } else {
-      activeItem = VANCHORS_MAP[poolAddress].fungibleTokenName;
+  const breadCrumbs = useMemo<BreadcrumbType[]>(() => {
+    // check if current path is /pool/<poolAddress>
+    if (parts.length === 3 && parts[0] === '' && parts[1] === 'pool') {
+      const poolAddress = parts[2];
```

Comment:
I believe it would be more prudent to use a more reliable method than array syntax `[]` to obtain the pool address. If the structure of the pathname changes, the current implementation could fail. In this instance, a regular expression (regex) might be a more robust solution.